cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -233,10 +235,13 @@ pub enum SseOpcode {
     Maxsd,
     Minss,
     Minsd,
+    Movaps,
```

Comment:
Can you teach the register allocator that Movaps is a move instruction, by adding it to the list of instructions matched in `is_move` in inst/mod.rs? (This is what enables register coalescing)